plugin/plaintextmodel.cpp

```diff
@@ -116,10 +119,18 @@ void PlainTextModel::setSource(const QUrl &source)
                 qmlInfo(this) << "Can't open " << m_source << ": " << m_file.errorString();
                 m_status = Error;
             } else {
+                qint64 limit = 5*1024*1024;
+                qint64 limitSize = qMin(m_file.size(), limit);
+
+                uchar * res = m_file.map(0, limitSize);
```

Comment:
Where does it get unmapped?

Reply:
> Where does it get unmapped?

nowhere. As @dcaliste told, mapping is better for reading big data than file.readAll()

Reply:
Anyway you are right. I'll do it

Reply:
Done

Reply:
Ok

Reply:
> I would also prefer to use read-only mapping

I think it's already mapped read-only because the `m_file` member has been set read-only itself around line 115.